distributed/scheduler.py

```diff
@@ -3980,6 +3980,8 @@ async def log_errors(func):
         self.stop()
         await super().close()
 
+        self.monitor.close()
```

Comment:
Can we centralize this a bit higher up in the class hierarchy? My guess is this would make more sense in the `Server` class

Reply:
Oh, good tip!